apollo-router/Cargo.toml

```diff
@@ -184,11 +184,11 @@ tower-service = "0.3.2"
 # https://github.com/apollographql/router/pull/1509.  A comment which exists
 # there (and above around the `opentelemetry` packages) should this ever change.
 # There is also a note below on the `^tracing` dev-dependencies!
-tracing = "=0.1.34"
-tracing-core = "=0.1.26"
+tracing = "=0.1.35"
```

Comment:
should we remove that condition on the tracing version now that the insta failures appear resolved?

Reply:
That would make sense yes